packages/atomic/src/components/search/atomic-search-box/atomic-search-box.pcss

```diff
@@ -11,6 +11,10 @@
   @apply absolute w-5/6 h-5/6 bg-background rounded-full;
 }
 
+[part~='instant-results-item'] {
```

Comment:
Should this instead be using a value from Tailwind ? (ie: `p-6` or similar)

That way if someones wants to theme the "compactness" (I'm not sure how to phrase it better 😄) of their UI, it will also applies to padding for instant result.

Reply:
yeah, that's a good shout!